hopp/eco/hybrid_system.py

```diff
@@ -0,0 +1,435 @@
+# general imports
+from distutils.command.config import config
+import os
+import numpy as np
+import pandas as pd
+pd.options.mode.chained_assignment = None  # default='warn'
+import numpy_financial as npf
+import sys
+from scipy import optimize
+
+# visualization imports
+import matplotlib.pyplot as plt
+
+# HOPP imports 
+import hopp.eco.electrolyzer as he_elec
+import hopp.eco.finance as he_fin
+import hopp.eco.hopp_mgmt as he_hopp
+import hopp.eco.utilities as he_util
+import hopp.eco.hydrogen_mgmt as he_h2
+
+# set up function to run base line case
+def run_simulation(filename_turbine_config, filename_orbit_config, filename_floris_config, electrolyzer_rating=None, plant_size=None, verbose=False, show_plots=False, save_plots=False, use_profast=True, 
+                   storage_type=None, incentive_option=1, plant_design_scenario=1, output_level=1, grid_connection=None):
+
+    # load inputs as needed
+    plant_config, turbine_config, wind_resource, floris_config = he_util.get_inputs(filename_orbit_config=filename_orbit_config, filename_floris_config=filename_floris_config , filename_turbine_config=filename_turbine_config, verbose=verbose, show_plots=show_plots, save_plots=save_plots)
+
+    if electrolyzer_rating != None:
+        plant_config["electrolyzer"]["rating"] = electrolyzer_rating
+
+    if grid_connection != None:
+        plant_config["project_parameters"]["grid_connection"] = grid_connection
+
+    if storage_type != None:
+        plant_config["h2_storage"]["type"] = storage_type
+
+    if plant_size != None:
+        plant_config["plant"]["capacity"] = plant_size
+        plant_config["plant"]["num_turbines"] = int(plant_size/turbine_config["turbine_rating"])
+        print(plant_config["plant"]["num_turbines"])
+
+    # 7 scenarios, 3 discrete variables
+    design_scenario = plant_config["plant_design"]["scenario%s" %(plant_design_scenario)]
```

Comment:
Would be nice to check if design_scenario selected is feasible 
i.e.
```
if design_scenario['transportation'] == 'hvdc+pipeline' or 'pipeline' 
and
design_scenario['electrolyzer_location'] == 'onshore':
         raise Exception(`plant_design` `scenario` chosen not possible)
```

Reply:
I think a warning may be more appropriate since this scenario may be desired for some reason. I don't think this is necessary at this time though.